src/software/ai/hl/stp/tactic/penalty_kick_tactic/penalty_kick_tactic_fsm.h

```diff
@@ -0,0 +1,299 @@
+#pragma once
+
+#include "software/ai/hl/stp/action/move_action.h"
+#include "software/ai/hl/stp/tactic/dribble/dribble_fsm.h"
+#include "software/ai/hl/stp/tactic/get_behind_ball/get_behind_ball_fsm.h"
+#include "software/ai/hl/stp/tactic/kick/kick_fsm.h"
+#include "software/ai/hl/stp/tactic/move/move_fsm.h"
+#include "software/ai/hl/stp/tactic/tactic.h"
+#include "software/geom/algorithms/closest_point.h"
+#include "software/geom/algorithms/intersection.h"
+
+struct PenaltyKickTacticFSM
+{
+    class ApproachBallState;
+    class ApproachKeeperState;
+    class InitialState;
+    class ShootState;
```

Comment:
seems like only InitialState is being used?

Reply:
forgot to remove them from an earlier version